src/components/inputs/Input/index.js

```diff
@@ -74,8 +74,8 @@ const StyledInput = styled.input`
     }
 
     &:focus {
-        box-shadow: rgb(214, 215, 215) 0px 0px 6px 0px;
-        border-color: rgb(169, 169, 169);
+        box-shadow: ${colors.INPUT_FOCUS} 0px 0px 6px 0px;
+        border-color: ${props => (props.border || colors.LABEL_COLOR)};
```

Comment:
It seems weird to me that focus selector in Input uses variable named  `LABEL_COLOR` for its border color. I do think we should store this color in separate variable as we already did in next wallet.

Reply:
Agree, so i made them to 3 separate variables:
INPUT_BORDER
INPUT_FOCUS_SHADOW
INPUT_FOCUS_BORDER

Reply:
awesome!